### DISCUSSION TITLE
Ability to pin private repos to an organization

### DISCUSSION THREAD

**Discussion by jeffpapp** [community/community#4917] [Enterprise] [answered]
It would be nice if we could pin private repos to our organization profile that would be visible to users who have access to them.  We don't really have any public repos that make sense to pin so it's currently showing popular public repos which doesn't make sense for our organization.

**Comment by cehenriques** [comment]
I'm also surprised that I cannot pin private repositories on an Enterprise organization. 
As an Enterprise user, where 100% of our projects are private, there's no sense in just allowing pinning public repositories.
Pinning private repos would allow our users to quickly access the most relevant repos.

**Reply by mattehicks** [reply]
Highly agree. They need to ADD THIS!!!!!!!

**Reply by andlewis** [reply]
I'm also waiting on this enhancement.

**Comment by jeffpapp** [comment, marked as answer]
This is live now 🎉 

https://github.blog/changelog/2022-04-20-organization-profile-updates-member-only-readmes-and-pinned-private-repositories/

**Reply by mattehicks** [reply]
Woohoo!!!
